portfolio/src/main/webapp/script.js

```diff
@@ -153,20 +154,29 @@ function getMessageFromServer() {
         });
       });
 }
-/** Clears all comments*/
+/** Clears all comments by removing them from the datastore*/
 function clearComments() {
   const request = new Request('\delete-data', {method: 'Post'});
   fetch(request).then(getMessageFromServer);
 }
-/** Gets the login status*/
+/** Gets the user object from the server which
```

Comment:
nit: this time the comment is too verbose. Now it's explaining all the fields of the JSON object which are not needed to understand what the function is doing. If you add a field to the JSON, for example, this comment will then be outdated and you'll have to update it, but not the function itself - that's a sign that the comment is not needed here. Additionally, you're explaining what processLoginStatus() is doing in this comment, but you could explain it in the comment for processLoginStatus.

The comment for a function should simply describe what the function does, including anything of note that is needed to know about what happens when we call the function, e.g. if it issues a request, or if it assumes a specific input, or if changes the state, etc. Focus more on the "why" and less on the "how".
In this case this function is pretty much boilerplate so we can't focus much on the "why", and the function is already pretty self-documenting so there is little need to add comments for it. But if the linter wants a comment - we can either disable the linter warning for this function, or write a simpler comment, for example:
`/** Sends a request to the '/login' URL to get a JSON response containing the current logged-in user, and calls processLoginStatus(user) on this user. */.`

Note that the comment doesn't tell us anything new (`fetch` implies a HTTP request, `response.json()` shows that we're getting a JSON object, etc.), but at least it's shorter and more to the point.

If you want to document what each field of the JSON is, the proper place should be in the servlet where the JSON itself is returned, as a comment in the DoGet() method which returns the object. And in processLoginStatus(), where you are actually reading the JSON fields, you can mention that the object is defined in LoginStatusServlet.java, so that any readers of the code can go to that class to see what the fields are, or briefly explain the fields in the @param part of the comment.